stores/sql/mysql/migrations/main/migration_00026_autopilot_state_1.sql

```diff
@@ -0,0 +1,29 @@
+-- remove references to autopilots table
+ALTER TABLE host_checks DROP FOREIGN KEY fk_host_checks_autopilot;
+ALTER TABLE host_checks DROP COLUMN db_autopilot_id;
+ALTER TABLE host_checks DROP INDEX idx_host_checks_id;
+ALTER TABLE host_checks ADD UNIQUE INDEX idx_host_checks_id (db_host_id);
+
+-- create autopilot_state table & insert blank state object
+CREATE TABLE `autopilot_state` (
```

Comment:
Maybe `autopilot` is sufficient. `autopilot_state` feels a bit redundant after all. It's a database so it obviously stores state.